src/ui/flutter_app/lib/mill/src/types.dart

```diff
@@ -157,6 +158,22 @@ extension PieceColorExtension on PieceColor {
         return S.of(context).draw;
       case PieceColor.ban:
       case PieceColor.nobody:
+        throw Exception("Player has no name");
```

Comment:
At least when the value is `ban`, an exception should not be thrown.

Reply:
Iirc there was an `assert(false)` in exactly the same place. 

In this case maybe the string is nit that accurate but it still applies. 

As the exceptions are not runtime related we should probably change them to Errors anyways